Content/posts/supporting-swift-59.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2023-06-12 12:00
+title: Supporting Swift 5.9 Betas
+description: How could we improve on adding support for a new Swift version after just one week of it being available? We could add support for Swift 5.9 one week after the release of the first beta version!
+---
+
+Back in April, we were [proud to announce that we supported Swift 5.8 within one week of the official release](https://blog.swiftpackageindex.com/posts/supporting-swift-58/), adding it to our compatibility matrix so you could know whether the package you were considering would work with the latest version of Swift.
+
+How could we improve on adding support for a new Swift version after just one week of it being available? We could add support for Swift 5.9 one week after the release of the _first beta version_!
+
+So, that’s the announcement! The Swift Package Index **now supports Swift 5.9** packages, including macros, the new syntax, and [everything else Swift 5.9 brings](https://www.hackingwithswift.com/articles/258/whats-new-in-swift-5-9)!
```

Comment:
We're not actually showing anything about macros anywhere (and search has an issue: https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/2425) so we might not want to call that out yet. (Although I hope the fix isn't going to be too involved.)

Reply:
I just added the issue for displaying macros so I'll add a link to that and make it clear it's not yet available. I don't think the search issue needs calling out specifically as it only affects filtering, if I understand it correctly?

Reply:
Yea, it's just the `product:macro` filter